api/rest/client.go

```diff
@@ -58,7 +58,7 @@ func (c *Client) NewRequest(method string, u *url.URL, payload interface{}) (req
 	}
 
 	req.Header.Set("Circle-Token", c.circleToken)
-	req.Header.Set("Accept-Type", "application/json")
+	req.Header.Set("Accept", "application/json")
```

Comment:
I wrote a description of this change in the `Consideration` section of the PR description.